### PR TITLE
[4.4] Set permission for add-module-to-dashboard button

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #41624 .

PR #37268 set wrong permission for the "Add module to dahboard" Button on cpanel. 
This was caused because a user had a special use case - descibed in https://github.com/joomla/joomla-cms/issues/37267

### Summary of Changes
This PR sets the permission back to core.manage on com_modules. 

Therefore the issue https://github.com/joomla/joomla-cms/issues/37267 is unresolved again. 
This special use case needs an other solution. Maybe an override of cpanel. 

### Testing Instructions
You need a user in usergroup manager. Login as manager - you see the button "Add Module to the dashboard". 
Activate. 
An Error message is displayed.

Apply the patch - no button for usergroup "manager". 

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
Could you explain what the problem is as it is not clear from your post

**Comment by chmst** [comment]
Users in group "manager"" are not allowed to work on com_modules. log in as manager and the Menu (content) is: 
grafik

If the user now activates the button and has not the permission, it gives an error. 
grafik

grafik

File: administrator/components/com_cpanel/tmpl/cpanel/default.php [hunk 1]
```diff
@@ -61,7 +61,7 @@
             echo ModuleHelper::renderModule($module, ['style' => 'well']);
         }
         ?>
-        <?php if ($user->authorise('core.create', 'com_modules')) : ?>
+        <?php if ($user->authorise('core.manage', 'com_modules')) : ?>
             <div class="module-wrapper">
```
Comment (Sieger66) [inline administrator/components/com_cpanel/tmpl/cpanel/default.php:65]: Why not:
`<?php if (($user->authorise('core.manage', 'com_modules')) and ($user->authorise('core.create', 'com_modules'))) : ?>`

Comment (chmst) [reply administrator/components/com_cpanel/tmpl/cpanel/default.php:65]: no, thank you, I hava a better solution

**Comment by chmst** [comment]
> Therefore the issue https://github.com/joomla/joomla-cms/issues/37267 is unresolved again.
resolved now also for this case.

**Comment by carlitorweb** [comment]
I have tested this item :white_check_mark: successfully on 2eef0188894b81e0928727ab3718da34dcb25af3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42467">issues.joomla.org/tracker/joomla-cms/42467</a>.</sub>

**Comment by carlitorweb** [comment]
In the case of #37267 will still work fine and no need any change after apply this PR

**Comment by carlitorweb** [comment]
@chmst delete from summary _at least_ this part: "**This PR sets the permission back to core.manage on com_modules.**"

**Comment by Sieger66** [comment]
I have tested this item :red_circle: unsuccessfully on 2eef0188894b81e0928727ab3718da34dcb25af3<br /><br />With this PR at this point all users in the usergroup "Administrator"
see not the "Add Module to the dashboard" button.

No button for usergroup "Administrator" only for "Super Users".

Alternative I have test it with: 
<?php if (($user->authorise('core.admin', 'com_modules')) and ($user->authorise('core.create', 'com_modules'))) : ?>
This works for me as aspected.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42467">issues.joomla.org/tracker/joomla-cms/42467</a>.</sub>

**Comment by Sieger66** [comment]
I have tested this item :white_check_mark: successfully on 39a1c855ee118007703a1b71c463ce8ac49915a0<br /><br />Works as aspected.
> @chmst delete from summary _at least_ this part: "**This PR sets the permission back to core.manage on com_modules.**"
@carlitorweb : Now we need a second test.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42467">issues.joomla.org/tracker/joomla-cms/42467</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 39a1c855ee118007703a1b71c463ce8ac49915a0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42467">issues.joomla.org/tracker/joomla-cms/42467</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42467">issues.joomla.org/tracker/joomla-cms/42467</a>.</sub>

**Comment by MacJoom** [comment]
Thank you!

**Pull request merged**